### DISCUSSION TITLE
ERC20 Tokens deployed from Ganache wont display in Metamask

### DISCUSSION THREAD

**Discussion by gigamike** [ConsenSys-archive/ganache#3877] [General]
Hi,

I'm using Ganache in my local and connecting it to my Metamask. Why is it the tokens from an ERC20 smart contract wont display in Metamask i.e. https://gyazo.com/2a622870b273783def9a25da66b68d34. Any ideas?

Thanks in advance

**Comment by davidmurdoch** [comment]
What version of Ganache?

**Comment by gigamike** [comment]
Hi @davidmurdoch ,

Version 2.5.4 (2.5.4.1367) for Mac

Thanks

**Reply by davidmurdoch** [reply]
We are currently working on updating Ganache UI to use latest version of the Ganache package (this repo) which might fix this. If you are able to run Ganache CLI it might work for you. Instructions here: https://github.com/trufflesuite/ganache#command-line-use

If using Ganache CLI (currently at v7.5.0) doesn't fix it for you, it may be that MetaMask uses an external service for this token screen, so it might not be something under Ganache's control.

**Comment by gigamike** [comment]
Thank you so much @davidmurdoch
